classes/classes/Items/Consumables/SweetGossamer.as

```diff
@@ -235,12 +235,15 @@ package classes.Items.Consumables
 				changes++;
 			}
 			//eyes!
-			if (player.hasPlainSkin() && (player.face.type !== Face.SPIDER_FANGS || player.face.type !== Face.HUMAN) && player.eyes.type == Eyes.HUMAN && rand(4) === 0 && changes < changeLimit) {
```

Comment:
Removed `&& (player.face.type !== Face.SPIDER_FANGS || player.face.type !== Face.HUMAN)` because that would always be true and thus is was pointless.